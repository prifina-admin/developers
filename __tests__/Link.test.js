import React, {createRef} from "react";
import renderer from "react-test-renderer";
import ThemeProvider from "../src/theme/ThemeProvider";
import Link from "../src/Link";

test("Link renders correctly", () => {
    const tree = renderer.create(
        <ThemeProvider>
            <Link href={"#"} target={"_blank"}>
                Something
            </Link>
        </ThemeProvider>).toJSON();
    expect(tree).toMatchSnapshot();
});

