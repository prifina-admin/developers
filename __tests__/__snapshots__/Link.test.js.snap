// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Link renders correctly 1`] = `
<div
  className="ThemeProvider__Base-sc-1rk52n4-0 emGfPA"
>
  <a
    className="Link__LinkElement-ka8deg-0 gOKVs"
    href="#"
    rel="noopener"
    target="_blank"
  >
    Something
  </a>
</div>
`;
